LBHackney-IT/lbh-fss-portal-frontend
# added access permissions to users and analytics pages, restriced visi…

**Description** 

Restrict access to /users and /analytics, and visible of users and analytics menu links just to those with 'hackney_viewer' or 'hackney_admin' roles, users with 'vsco_contributer' role should not be able to access these pages are view these menu links 

**Test instructions**

- pull this branch
- navigate to mock-api/users/mockUsers.json and temporarily delete all users except one, update the user roles with varying combinations of 'hackney_viewer',  'hackney_admin' and 'vsco_contributer' (will need to restart mock api `yarn mock-api` between changes)
- check expected behaviour by viewing menu links and trying to access /users and /analytics  (see screenshot below of expected view if user does not have permissions to access either /user or /analytics) 

![image](https://user-images.githubusercontent.com/69238887/90890291-97940c00-e3b1-11ea-9ffa-5a85c7bc4799.png)

**Ticket**

https://app.clickup.com/t/9e5u9e
